src/modules/tasks.js

```diff
@@ -0,0 +1,38 @@
+export const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function storeTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function checkTasks() {
+  const lists = document.querySelector('.lists');
+
+  tasks.sort((a, b) => a.index - b.index);
+
+  tasks.forEach((task) => {
+    lists.insertAdjacentHTML('beforeend', `<li class="task-item" contentEditable = "false"><input type="checkbox">${task.description}<i id="task-btn" class="fa-solid fa-ellipsis-vertical"></i></li>`);
+  });
+}
+
+export function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(task);
+  storeTasks();
+}
+
+export function removeTask(index) {
+  tasks.splice(index, 1);
+  for (let i = index; i < tasks.length; i += 1) {
+    tasks[i].index = i;
+  }
+  storeTasks();
+}
```

Comment:
- [ ] Good job implementing this functionality, but kindly make changes so that users can delete tasks of their choice, currently only the items which were added first are getting deleted.
- [ ] One of the project requirements is that, when an item is deleted, the index of the array in the local storage should be updated, for example, if we have 3 items and we delete 1, the index of the second element should be updated to 2. Let's fix this issue as well.

![update](https://github.com/RicardoEgas/To-Do-List/assets/53124548/22293d33-fe78-479c-aed7-3eb07c00a3bd)